bifacial_radiance/mismatch.py

```diff
@@ -179,7 +179,7 @@ def mismatch_fit3(data):
     Returns
     -------
     fit3 :  Float or pd.Series
-        Returns mismatch values for each module
+        Returns mismatch values for each module in percentage [%].
     
     Equation: 1/(n^2*Gavg)*Sum Sum (abs(G_i - G_j))
     ## Note: starting with Pandas 1.0.0 this function will not work on Series objects.
```

Comment:
Please include this deprecation message:
    ## Note: This fit value is deprecated in favor of Fit2 which has better agreement
    ##       with the experimental data.

Reply:
There it is with a Sphinx directive.

I haven't seen any deprecation module so deprecation tests haven't been added.